packages/core/src/modules/proofs/protocol/v2/__tests__/presentation-exchange-presentation-format-data.e2e.test.ts

```diff
@@ -98,6 +100,7 @@ describe('Present Proof', () => {
     const aliceProofExchangeRecordPromise = waitForProofExchangeRecord(aliceAgent, {
       threadId: aliceProofExchangeRecord.threadId,
       state: ProofState.Done,
+      timeoutMs: 200000, // Temporary I have increased timeout as, verify presentation takes time to fetch the data from documentLoader
```

Comment:
Is this not using the local document loader? 200 seconds is getting crazy...

Reply:
I reckon it could be because I am having to issue multple credentials, each of which has to be signed.

This takes an age.

Reply:
Can you do some debugging on what it is that is taking so long? The tests already take too long to run, and I don't want to keep increasing the timer. There must be something we can do to speed it up.

Reply:
sure

Reply:
I did some performance testing and the time taken for each of the e2e tests was between 8 and 12 seconds, so there doesn't seem to be a problem here. I have removed the timeouts.

Reply:
Okay, perfect 👍